.github/workflows/CompatHelper.yml

```diff
@@ -2,7 +2,7 @@ name: CompatHelper
 
 on:
   schedule:
-    - cron: '* * * * 0'
+    - cron: '0 0 * * 0'
```

Comment:
Isn't this once per day?

Reply:
midnight on sunday, so once per week.  https://crontab.guru/#0_0_*_*_0

Reply:
it's a shame that both tagbot and compathelper poll.  would be nice if they could be re-engineered to use interrupts.